server/custom_typings/express/index.d.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare namespace Express {
+  interface Request {
+    decoded: import('firebase-admin').auth.DecodedIdToken;
```

Comment:
decoded -> firebaseDecodedIdToken?

Reply:
I think the convention is `req.decoded` when you put something you decoded from a token.